src/lib/y2_alternatives/dialog/alternative.rb

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2016 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require "yast"
+require "ui/dialog"
+
+Yast.import "UI"
+Yast.import "Label"
+
+module Y2Alternatives
+  module Dialog
+    # Dialog for displaying possible Choices for one particular Alternative.
+    class AlternativeDialog < UI::Dialog
```

Comment:
As already commented, this name is redundant. https://github.com/yast/yast-update-alternatives/pull/23#discussion_r71298083